go/obscuronode/enclave/storage.go

```diff
@@ -278,7 +278,8 @@ func (s *storageImpl) IsBlockAncestor(block *types.Block, maybeAncestor obscuroc
 }
 
 func (s *storageImpl) assertSecretAvailable() {
-	if s.FetchSecret() == nil {
-		panic("Enclave not initialized")
-	}
+	// TODO uncomment this
```

Comment:
@otherview Can you expand the comment to say what will cause this to be re-enabled?

Reply:
Fixed :)